src/extensions/default/Brackets-InteractiveLinter/main.js

```diff
@@ -20,7 +20,18 @@ define(function (require, exports, module) {
         preferences        = PreferencesManager.getExtensionPrefs("interactive-linter"),
         linterManager      = require("linterManager"),
         pluginManager      = require("pluginManager");
-
+        
+	var defaultPreferences = { 
+    	"enabled": false,
+    	"javascript" : ["eslint"] 
+    };
+
+        function loadPreferences(){
+            preferences.definePreference("enabled", "boolean", defaultPreferences.enabled);
+            preferences.definePreference("javascript", "array", defaultPreferences.javascript); 
```

Comment:
I did some debugging, and it's crashing when it tries to redefine this preference when the `eslint` provider is loaded.  Remove this line (31) and line 32 below.  In fact, just change lines 24-33 to this:

```js
function addEnabledPref() {
    // We disable this by default
    preferences.definePreference("enabled", "boolean", false);
}
```